script.module.clouddrive.common/addon.xml

```diff
@@ -16,7 +16,14 @@
 		<license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```

Comment:
I'd be nice if you can change the `license` tag to a spdx identifier at some point in the future. See https://spdx.org/licenses/

in this case it should be `GPL-3.0-only` or `GPL-3.0-or-later` depending on the license

Reply:
Sounds good. Will be there in the next PR.